netjsonconfig/backends/base/backend.py

```diff
@@ -49,6 +50,17 @@ def __init__(self, config=None, native=None, templates=None, context=None):
             raise ValueError('Expecting either config or native argument to be '
                              'passed during the initialization of the backend')
 
+    @classmethod
+    def add_plugins(cls, plugins):
+        for plugin in plugins:
+            plugin.schema.pop('$schema', None)
+            if getattr(cls, 'schema'):
```

Comment:
as I was saying, we expect backend to always have a `schema`, therefore if `schema` is not present we should "fail early and loud"